### PR TITLE
Add macOS  to build matrix

### PR DESCRIPTION
**Pull request by djphoenix**
Some of tests are fails as mentioned in #1264 and #1353. I haven't already find out why.

### CODE REVIEW

File: tests/language-phonemes.test [hunk 1]
```diff
@@ -89,8 +89,8 @@ test_phwav la 080bd53c20991eae7baec73b8c735eacc8aae076 "ma na Na pa p<h>a ba ta
...
-test_phwav ltg 9e0ee2a095cd074860c3db7aa89b2011ffb38ee8 "ma m;a na n;a pa p;a ta t;a ka k;a ba b;a da d;a ga g;a tsa ts;a tSa tS;a dza dz;a dZa dZ;a fa f;a sa s;a Sa S;a xa x;a va v;a za z;a Za Z;a la l;a ra r;a ja _:_ m@ ma mA ma: me mE me: mee meA mi mI mi: mo mO mo: mu mU mu: mw mW mai mei mau muo moi mui mie maU meU moU maI meI"
-test_phwav lv d1a84f9881e4819b46f907723184587e28e885d9 "ma na n^a Na pa ba ta da ca Ja ka ga tsa dza Dz\`a tSa dZa DZ\`a fa va sa za Sa Za xa ha ja la l^a Ra ra _:_ mi my mu mE me mo ma mi: my: mu: mE: me: mo: ma: mai mau mei mie miu mui muo muo\` moi"
+# test_phwav ltg 9e0ee2a095cd074860c3db7aa89b2011ffb38ee8 "ma m;a na n;a pa p;a ta t;a ka k;a ba b;a da d;a ga g;a tsa ts;a tSa tS;a dza dz;a dZa dZ;a fa f;a sa s;a Sa S;a xa x;a va v;a za z;a Za Z;a la l;a ra r;a ja _:_ m@ ma mA ma: me mE me: mee meA mi mI mi: mo mO mo: mu mU mu: mw mW mai mei mau muo moi mui mie maU meU moU maI meI"
+# test_phwav lv d1a84f9881e4819b46f907723184587e28e885d9 "ma na n^a Na pa ba ta da ca Ja ka ga tsa dza Dz\`a tSa dZa DZ\`a fa va sa za Sa Za xa ha ja la l^a Ra ra _:_ mi my mu mE me mo ma mi: my: mu: mE: me: mo: ma: mai mau mei mie miu mui muo muo\` moi"
```
Comment (sthibaul) [inline tests/language-phonemes.test:93]: Rather than disabling the test, could you add something like an if [ "$(uname -s)" != macos ] ?

**Comment by sthibaul** [comment]
@alex19EP We need to have something that reminds us to re-enable the disabled tests. It's really fishy that it behaves differently on macos.

**Comment by djphoenix** [comment]
@sthibaul this is my next target to find out the root cause of differencies.

**Comment by alex19EP** [comment]
@sthibaul sorry. I didn't see your comments. I agree.

**Pull request merged**